### DISCUSSION TITLE
Future of getServerSideProps and Page Router in general

### DISCUSSION THREAD

**Discussion by eugeniosegala** [vercel/next.js#56689] [App Router]
Hello!

Given then Next.js is going to be primarily developed around the pattern of the new "App Router", but what is the future of "getServerSideProps" and the "Page Router" paradigm in general?

Are there any plans to deprecate it/remove it in the future? Or will it still be supported for long time?

**Comment by eugeniosegala** [comment]
Any indications would be really appreciated 🙏

**Reply by 4ydx** [reply]
I would also like to know this.  The page router is my preference because it doesn't force a node backend.
